runtime/compiler/z/codegen/J9BCDTreeEvaluator.cpp

```diff
@@ -2701,8 +2701,8 @@ J9::Z::TreeEvaluator::pdcmpVectorEvaluatorHelper(TR::Node *node, TR::CodeGenerat
    generateRRInstruction(cg, TR::Compiler->target.is64Bit() ? TR::InstOpCode::XGR : TR::InstOpCode::XR, node, resultReg, resultReg);
    generateLoad32BitConstant(cg, node, 1, resultReg, true);
 
-   TR::RegisterDependencyConditions* deps = new(cg->trHeapMemory()) TR::RegisterDependencyConditions(0, 1, cg);
-   TR::LabelSymbol* doneCompareLabel = TR::LabelSymbol::create(cg->trHeapMemory(), cg);
+   TR::RegisterDependencyConditions* deps = new(cg->trHeapMemory()) TR::RegisterDependencyConditions(0, 2, cg);
```

Comment:
Why 2 post dependencies?  I only see one call of `addPostCondition`.

Reply:
Is it because  `generateLoad32BitConstant` can potentially allocate another register?

Reply:
The implicit dependency changes in APIs like `generateLoad32BitConstant()` can be tricky.. How do we know the number of dependencies they are adding?

Reply:
> Is it because `generateLoad32BitConstant` can potentially allocate another register?

Yes, this is precisely it.